public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -2,6 +2,7 @@ import { css } from '@emotion/css';
 import React, { PureComponent } from 'react';
 
 import { FeatureState, SelectableValue } from '@grafana/data';
+import { getThemesList } from '@grafana/data/src/themes/registry';
```

Comment:
think we should expose this properly from `@grafana/data` so we don't have to do a nested import

Reply:
see also the `@grafana/ui/src/unstable` approach we have. Not saying we should also do that here, but there is precedent for this...